examples/guides/services/healthcare-service/tests/clemency-hospital-service-test.bal

```diff
@@ -0,0 +1,547 @@
+                    },
+                    {
+                        "statusCode":400,
+                        "responseMessage":"Doctor T Uyanage is not available in clemency medical center"
+                    }
+                ],
+                [
+                   {  
+                        "patient":{  
+                            "name":"Little John",
+                            "dob":"2018-04-29",
+                            "ssn":"",
+                            "address":"California",
+                            "phone":"",
+                            "email":""
+                        },
+                        "doctor":"cailen cooper",
+                        "hospital":"clemency medical center",
+                        "appointmentDate":"2019-12-02"
+                    },
+                    {
+                        "statusCode":200,
+                        "appoinmentNumber":true,
+                        "doctorAvailibility":true,
+                        "doctorFee":true
+                    } 
+                ]
+            ]; 
+        }
+
+# Description: This test scenario verifies if details of the reserved appoinment can be retrived. 
+# TC004 - Verify if appoinment details can be retrieved successfully by providing a valid appointment number.
+# TC005 - Verify if an error occurs by providing an invalid appointment number.  
+#
+# + dataset - dataset Parameter Description
+@test:Config
+{
+    dataProvider: "testGetAppointmentClemencyDataProvider",
+    dependsOn: ["testReserveAppointment"]
+}
+
+
+function testGetAppointmentClemency(json dataset)
+{
+    var expectedAppointmentNumber = dataset.appointmentNumber;
+    var expectedDoctorName = dataset.doctorName;
+    var expectedAppointmentDate = dataset.appointmentDate;
+
+    var response = clientEPclemency->get("/appointments/"+expectedAppointmentNumber.toString());
+    if(response is http:Response)
+    {
+        if(expectedAppointmentNumber == 20000000)
+        {
+            var responsePayload = response.getTextPayload();
+            test:assertEquals(responsePayload, "Invalid appointment number.", msg = "Assertion Failed!");
+        }
+        else
+        {
+            var responsePayload = response.getJsonPayload();
+            if(responsePayload is json)
+            {
+                var responseAppointmentNumber = responsePayload.appointmentNumber;
+                var responseDoctorName = responsePayload.doctor.name;
+                var responseAppoinmentDate = responsePayload.appointmentDate;
+
+                test:assertEquals(responseAppointmentNumber, expectedAppointmentNumber, msg = "Appointment number is not as expected!");
+                test:assertEquals(responseDoctorName, expectedDoctorName, msg = "Doctor's name is not as expected!");
+                test:assertEquals(responseAppoinmentDate, expectedAppointmentDate, msg = "Appointment date is not as expected!");
```

Comment:
These lines are too long. Max line length is 120 characters.

Reply:
Done.